spec/log-window-tests.js

```diff
@@ -26,20 +26,33 @@ describe('Log window', () => {
     mockDocument.createElement = (name) => document.createElement(name);
     document.body.appendChild(mockDocument);
 
+    chrome = {
```

Comment:
It seems like you're implicitly assigning this to a global, effectively window.chrome.

If you do that, please:
 1. Make it _explicit_ with `window.`
 2. Back up the original `window.chrome` in a local variable in `beforeAll()`
 3. Restore that value in `afterAll()`

It may not matter for the test suite as it stands today, but overwriting globals without backup/restore can lead to global namespace pollution between test suites in the future.

Reply:
I backed up the original `chrome.windows` in `beforeAll()` and restored it in `afterAll()`. And I didn't do the same thing for `chrome.runtime` since it is null in the selenium test environment.
```
let oldChromeWindows;
...
beforeAll(() => {
...
oldChromeWindows = chrome.windows;
..
});
afterAll(() => {
  chrome.windows = oldChromeWindows;
});
```